apis/cluster/serverpool.go

```diff
@@ -35,6 +35,7 @@ type ServerPool struct {
 	Image             string
 	Size              string
 	BootstrapScript   string
+	VPNScript         string
```

Comment:
I really like this idea! Its a lot more logical and makes it way easier when we add all kinds of other script. Less code, more functionality.